platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java

```diff
@@ -311,7 +311,7 @@ public void setLdapBindUserDn(String bindUserDn) {
     }
 
     public void setPassword(String password) {
-        LOGGER.trace("Setting password: {}", password);
+        LOGGER.trace("Setting password: [HIDDEN]");
```

Comment:
This was actually a request from the security guys when I discussed this type of issue with them.